examples/unconstrained/guest/src/main.rs

```diff
@@ -0,0 +1,35 @@
+#![no_main]
+
+use k256::FieldElement;
+
+zkm_zkvm::entrypoint!(main);
+
+pub fn main() {
+    const MODULUS: &str = "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141";
+    const NQR: FieldElement = FieldElement::from_u64(3);
+    let value = FieldElement::from_u64(1);
+    let (status, result) =
+        call_sqrt_hook(value.to_bytes().as_slice(), MODULUS, NQR.to_bytes().as_slice());
+    println!("status: {:?}", status);
```

Comment:
can we add an assert here to check the result is as expected?

Reply:
Actually, I don't know what the correct result should be. Currently, this example does not run successfully.

Reply:
This example is used to test `zkm_lib::unconstrained!`.

Reply:
Okay.  You can use sage to calculate it.  Or you can get the answer and then calculate the square, and verify  the result.  it's optional.

Reply:
ok